spec/docs/npq_participants_spec.rb

```diff
@@ -54,4 +55,81 @@
       end
     end
   end
+
+  path "/api/v1/participants/npq/{id}/withdraw" do
+    put "Withdrawn a participant from a course" do
+      operationId :npq_participants
+      tags "NPQ participant"
+      security [bearerAuth: []]
+      consumes "application/json"
+
+      request_body content: {
+        "application/json": {
+          "schema": {
+            "$ref": "#/components/schemas/NPQParticipantWithdrawalRequest",
+          },
+        },
+      }
+
+      parameter name: :id,
+                in: :path,
+                type: :string,
+                required: true,
+                example: "28c461ee-ffc0-4e56-96bd-788579a0ed75",
+                description: "The ID of the participant to withdraw"
+
+      parameter name: :params,
+                in: :body,
+                type: :object,
+                style: :deepObject,
+                required: true,
+                schema: {
+                  "$ref": "#/components/schemas/NPQParticipantWithdrawRequest",
```

Comment:
Question one: should it be the same schema as the body one above? `NPQParticipantWithdrawalRequest`? Probably yes, I think the schema in body is what's used in the documentation, and the schema in params is used by rswag to check the shape when running the test (I might be wrong).

Also, I don't see any `NPQParticipantWithdrawalRequest` or `NPQParticipantWithdrawRequest` yml files, which will likely cause something to break - most likely with an error saying that 'I could not load a schema' when trying to display swagger docs in some way.